frontends/web/src/routes/device/bitbox02/bitbox02.tsx

```diff
@@ -421,409 +423,378 @@ class BitBox02 extends Component<Props, State> {
-                                        i.e. the attestation warning at the top,
-                                        that is why added it there as well instead of
-                                        the password guesture. */}
-                                        {attestationResult === false ? (
-                                            <Status>
-                                                {t('bitbox02Wizard.attestationFailed')}
-                                            </Status>
-                                        ) : (
-                                            <PasswordEntry />
-                                        )}
-                                    </ViewContent>
-                                </View>
-                            ) : null }
-
-                            <Step
-                                key="failed-pairing"
-                                active={status === 'unpaired' || status === 'pairingFailed'}
-                                title={t('bitbox02Wizard.pairing.title')}>
-                                {
-                                    status === 'pairingFailed' && (
-                                        <Toast theme="warning">
-                                            <span>{t('bitbox02Wizard.pairing.failed')}</span>
-                                        </Toast>
-                                    )
-                                }
-                                <div className={[style.stepContext, status === 'pairingFailed' ? style.disabled : ''].join(' ')}>
-                                    <p>{t('bitbox02Wizard.stepUnpaired.verify')}</p>
-                                    <pre>{hash}</pre>
-                                    {
-                                            <div className={['buttons text-center', style.fullWidth].join(' ')}>
-                                                <Button
-                                                    primary
-                                                    onClick={() => this.channelVerify(true)}
-                                                    disabled={!deviceVerified}>
-                                                    {t('bitbox02Wizard.pairing.confirmButton')}
-                                                </Button>
-                                            </div>
-                                    }
-                                </div>
-                                <div className="text-center m-top-large">
-                                    <SwissMadeOpenSource large />
-                                </div>
-                            </Step>
-
-                            {
-                                !unlockOnly && (
-                                    <Step
-                                        key="uninitialized-pairing"
-                                        active={status === 'uninitialized' && appStatus === ''}
```

Comment:
active was once used as step active prop, but then the steps indicator got removed and this component was only used as container. The active prop did conditionally render the container and there is another condition around the container.

Changed View component with both conditions around the component.